src/Symfony/Component/HttpFoundation/Exception/JsonException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+final class JsonException extends \RuntimeException implements RequestExceptionInterface
```

Comment:
it would make sense to extend `JsonException` to me

Reply:
Agree, my fault to have proposed this.

Reply:
I mean, extend `\JsonException` instead of `RuntimeException`.
But removing final works also :)

Reply:
That does not work for php <7.3, isn't it?

Reply:
> That does not work for php <7.3, isn't it?

I guess the 7.3 polyfill would then need to be used